lib/oneview-sdk/resource/api1600/c7000/server_hardware.rb

```diff
@@ -0,0 +1,22 @@
+# (C) Copyright 2020 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+require_relative '../../api1200/c7000/server_hardware'
+
+module OneviewSDK
+  module API1600
+    module C7000
+      # Server Hardware resource implementation on API1600 C7000
+      class ServerHardware < OneviewSDK::API1200::C7000::ServerHardware
```

Comment:
type field has changed here..please add the initialize method to upgrade type field, otherwise POST will fail

Reply:
type is not an mandatory param for serverHardware resource. I confirmed the same after checking in python code repo

Reply:
yes you are right. Approved.